src/models/tools/tool-tile.ts

```diff
@@ -9,7 +9,7 @@ export const kDefaultMinWidth = 60;
 export const ToolTileModel = types
   .model("ToolTile", {
     // if not provided, will be generated
-    id: types.optional(types.string, () => uuid()),
+    id: types.optional(types.identifier, () => uuid()),
```

Comment:
I think we played around with this but ended up sticking with a `string`?

Reply:
I was actually glad to see this. We decided not to replace uses of `types.reference` with strings, but I think making this a `types.identifier` makes sense. Noah and I had used this technique on the Precipitating Change project, but something in the MST 3 documentation made me think it was no longer possible. Since we're not using references we're not taking full advantage of the use of `types.identifier`, but we still may be able to take advantage of some of them like the ability to look up instances by ID.